precompile/contracts/feemanager/contract_test.go

```diff
@@ -20,6 +20,17 @@ import (
 )
 
 var (
+	regressionBytes     = "8f10b58600000000000000000000000000000000000000000000000000000000017d78400000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000012a05f20000000000000000000000000000000000000000000000000000000000047868c0000000000000000000000000000000000000000000000000000000000000005400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001bc16d674ec800000000000000000000000000000000000000000000000000000de0b6b3a764000000000000000000000000000000000000000000000000000000000000"
+	regressionFeeConfig = commontype.FeeConfig{
+		GasLimit:                 big.NewInt(25000000),
+		TargetBlockRate:          2,
+		MinBaseFee:               big.NewInt(5000000000),
+		TargetGas:                big.NewInt(75000000),
+		BaseFeeChangeDenominator: big.NewInt(84),
+		MinBlockGasCost:          big.NewInt(0),
+		MaxBlockGasCost:          big.NewInt(2000000000000000000),
+		BlockGasCostStep:         big.NewInt(1000000000000000000),
+	}
```

Comment:
should we add some assertion that the regression bytes is the FeeConfig + some padding or derive the regression bytes by adding padding to the serialized regressionFeeConfig?
Also, do the padding bytes have to be 0 or do we allow any as this PR suggests?

Reply:
I derived the feeconfig from bytes and confirmed with the DFK that it was the requested config. So I don't think we need to continuously assert that these are equal. 

Yes it can be other than 0. But I think the common case is 0.